server/main.lua

```diff
@@ -1,83 +1,83 @@
 local QBCore = exports['qb-core']:GetCoreObject()
-local ActivePolice = 2  		--<< needed policemen to activate the mission
-local cashA = 250 				--<<how much minimum you can get from a robbery
-local cashB = 450				--<< how much maximum you can get from a robbery
-local ActivationCost = 500		--<< how much is the activation of the mission (clean from the bank)
-local ResetTimer = 2700 * 1000  --<< timer every how many missions you can do, default is 600 seconds
 local ActiveMission = 0
 
-RegisterServerEvent('AttackTransport:akceptujto')
-AddEventHandler('AttackTransport:akceptujto', function()
+RegisterNetEvent('truckrobbbery:AcceptMission')
+AddEventHandler('truckrobbbery:AcceptMission', function()
 	local copsOnDuty = 0
 	local _source = source
 	local xPlayer = QBCore.Functions.GetPlayer(_source)
 	local accountMoney = 0
 	accountMoney = xPlayer.PlayerData.money["bank"]
-if ActiveMission == 0 then
-	if accountMoney < ActivationCost then
-	TriggerClientEvent('QBCore:Notify', _source, "You need $"..ActivationCost.." in the bank to accept the mission")
-	else
-		for k, v in pairs(QBCore.Functions.GetPlayers()) do
-			local Player = QBCore.Functions.GetPlayer(v)
-			if Player ~= nil then
-				if (Player.PlayerData.job.name == "police" and Player.PlayerData.job.onduty) then
-					copsOnDuty = copsOnDuty + 1
+	if ActiveMission == 0 then
+		if accountMoney < Config.ActivationCost then
+			TriggerClientEvent('QBCore:Notify', _source, Lang:t('mission.Activation_cost', {ActivationCost = Config.ActivationCost}))
+		else
+			for k, v in pairs(QBCore.Functions.GetPlayers()) do
+				local Player = QBCore.Functions.GetPlayer(v)
+				if Player ~= nil then
+					if (Player.PlayerData.job.name == "police" and Player.PlayerData.job.onduty) then
+						copsOnDuty = copsOnDuty + 1
+					end
 				end
 			end
+			if copsOnDuty >= Config.ActivePolice then
+				TriggerClientEvent("truckrobbbery:StartMission", _source)
+				xPlayer.Functions.RemoveMoney('bank', Config.ActivationCost, "armored-truck")
+				HitTimer()
+			else
+				TriggerClientEvent('QBCore:Notify', _source, Lang:t('error.ActivePolice', {ActivePolice = Config.ActivePolice}))
+			end
 		end
-	if copsOnDuty >= ActivePolice then
-		TriggerClientEvent("AttackTransport:Pozwolwykonac", _source)
-		xPlayer.Functions.RemoveMoney('bank', ActivationCost, "armored-truck")
-
-		OdpalTimer()
-    else
-		TriggerClientEvent('QBCore:Notify', _source, 'Need at least '..ActivePolice.. ' SASP to activate the mission.')
-    end
+	else
+		TriggerClientEvent('QBCore:Notify', _source, Lang:t('error.alreadyactive'))
```

Comment:
locales/en.lua is AlreadyActive .. presuming locales are caps sensitive?
This should be Lang:t('error.AlreadyActive'))